apps/web/vibes/2px/components/button/index.tsx

```diff
@@ -1,64 +1,55 @@
-'use client'
+import { Slot } from '@radix-ui/react-slot'
+import { cva } from 'class-variance-authority'
 
-import Link from 'next/link'
-import { ReactNode, Ref, forwardRef } from 'react'
+import { cn } from '@/lib/utils'
 
-import clsx from 'clsx'
+import { LoadingIcon } from '../icons'
 
-export interface Props {
+export const buttonVariants = cva(
+  'rounded-[2.5rem] whitespace-nowrap flex items-center justify-center mx-auto font-body text-lg',
+  {
+    variants: {
+      variant: {
+        primary:
+          'bg-foreground text-background hover:bg-background hover:text-foreground hover:border-2 hover:border-foreground',
+        secondary: 'bg-background text-foreground border-2 border-foreground hover:border-dashed',
+      },
+    },
+    defaultVariants: {
+      variant: 'primary',
+    },
+  }
+)
```

Comment:
I think that in an effort to minimize dependencies we're going to stay away from `class-variance-authority`. I know @apledger particularly despises this library. We're thinking that the same pattern can be applied doing something like this:

```tsx
<div className={clsx('tailwind classes', { primary: 'some more for primary', secondary: 'some more for secondary' }[variant])} />
```